<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bulguekle</class>
 <widget class="QDialog" name="bulguekle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeni Bulgu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblHastatable">
         <property name="text">
          <string>Hasta Seç</string>
         </property>
         <property name="buddy">
          <cstring>HastatableWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="HastatableWidget"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblHastaBul">
         <property name="text">
          <string>Bul</string>
         </property>
         <property name="buddy">
          <cstring>leHastaBul</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leHastaBul"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="lblZiyaretBul">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Bul</string>
         </property>
         <property name="buddy">
          <cstring>leZiyaretBul</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="ZiyarettableWidget">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lblZiyarettable">
         <property name="text">
          <string>Ziyaret Seç</string>
         </property>
         <property name="buddy">
          <cstring>ZiyarettableWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leZiyaretBul">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblAciklama">
         <property name="text">
          <string>Açıklama</string>
         </property>
         <property name="buddy">
          <cstring>teAciklama</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="teAciklama"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="lblTip">
         <property name="text">
          <string>Tip</string>
         </property>
         <property name="buddy">
          <cstring>cbTip</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbTip">
         <item>
          <property name="text">
           <string>Semptom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fiziksel Muayene</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Labaratuvar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Görüntüleme</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblKaydedilmeZamani">
         <property name="text">
          <string>Kaydedilme Zamanı</string>
         </property>
         <property name="buddy">
          <cstring>dteKaydedilmeZamani</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="dteKaydedilmeZamani"/>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVazgec">
       <property name="text">
        <string>Vazgec</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnVazgec</sender>
   <signal>clicked()</signal>
   <receiver>bulguekle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
